File: CS/NetCore/net/13_Touch/ContextMenuScale.cs
namespace InputManWin13_Demo._13_Touch;

public partial class ContextMenuScale : InputManWin13_Demo.DemoBase_settings
{
    public ContextMenuScale()
    {
        InitializeComponent();
        InitializeForm();
    }

    private void InitializeForm()
    {
        // xmlファイルからデータ取得
        this.dataSet1.ReadXml(Path.Combine(Application.StartupPath, @"Resources\prefecture.xml"));
        gcComboBox1.DataSource = dataSet1.Tables[0];
        gcComboBox1.SelectedIndex = -1;

        for (int i = 1; i < gcComboBox1.ListColumns.Count; i++)
        {
            gcComboBox1.ListColumns[i].AutoWidth = true;
        }

        // xmlファイルからデータ取得（gcMaskedComboBox）
        this.dataSet2.ReadXml(Path.Combine(Application.StartupPath, @"Resources\pref.xml"));
        gcMaskedComboBox1.DataSource = dataSet2.Tables[0];
        gcMaskedComboBox1.SelectedIndex = -1;

        for (int i = 1; i < gcMaskedComboBox1.ListColumns.Count; i++)
        {
            gcMaskedComboBox1.ListColumns[i].AutoWidth = true;
        }

        // TouchContextMenuScaleプロパティ
        gcNumber2.ValueChanged += (s, e) => SetTouchContextMenuScale();
    }

    private void SetTouchContextMenuScale()
    {
        gcTextBox1.TouchContextMenuScale = (float)gcNumber2.Value.GetValueOrDefault();
        gcMask1.TouchContextMenuScale = (float)gcNumber2.Value.GetValueOrDefault();
        gcCharMask1.TouchContextMenuScale = (float)gcNumber2.Value.GetValueOrDefault();
        gcDateTime1.TouchContextMenuScale = (float)gcNumber2.Value.GetValueOrDefault();
        gcDate1.TouchContextMenuScale = (float)gcNumber2.Value.GetValueOrDefault();
        gcTime1.TouchContextMenuScale = (float)gcNumber2.Value.GetValueOrDefault();
        gcTimeSpan1.TouchContextMenuScale = (float)gcNumber2.Value.GetValueOrDefault();
        gcNumber1.TouchContextMenuScale = (float)gcNumber2.Value.GetValueOrDefault();
        gcComboBox1.TouchContextMenuScale = (float)gcNumber2.Value.GetValueOrDefault();
        gcMaskedComboBox1.TouchContextMenuScale = (float)gcNumber2.Value.GetValueOrDefault();
    }
}

File: VB/NetCore/net/13_Touch/ContextMenuScale.vb
Imports System.Windows.Forms

Namespace _13_Touch
    Partial Public Class ContextMenuScale
        Inherits InputManWin13_Demo.DemoBase_settings
        Public Sub New()
            InitializeComponent()
            InitializeForm()
        End Sub

        Private Sub InitializeForm()
            ' xmlファイルからデータ取得(GcComboBox)
            Dim filePath As System.String = Application.StartupPath
            Me.dataSet1.ReadXml(filePath & "\Resources\prefecture.xml")
            gcComboBox1.DataSource = dataSet1.Tables(0)
            gcComboBox1.SelectedIndex = -1

            For i As Integer = 1 To gcComboBox1.ListColumns.Count - 1
                gcComboBox1.ListColumns(i).AutoWidth = True
            Next

            ' xmlファイルからデータ取得(GcMaskedComboBox)
            Me.DataSet2.ReadXml(filePath & "\Resources\pref.xml")
            GcMaskedComboBox1.DataSource = DataSet2.Tables(0)
            GcMaskedComboBox1.SelectedIndex = -1

            For i As Integer = 1 To GcMaskedComboBox1.ListColumns.Count - 1
                GcMaskedComboBox1.ListColumns(i).AutoWidth = True
            Next
        End Sub

        Private Sub gcNumber2_ValueChanged(ByVal sender As Object, ByVal e As EventArgs) Handles gcNumber2.ValueChanged
            ' TouchContextMenuScaleプロパティ
            gcTextBox1.TouchContextMenuScale = CSng(gcNumber2.Value)
            gcMask1.TouchContextMenuScale = CSng(gcNumber2.Value)
            gcCharMask1.TouchContextMenuScale = CSng(gcNumber2.Value)
            gcDateTime1.TouchContextMenuScale = CSng(gcNumber2.Value)
            gcDate1.TouchContextMenuScale = CSng(gcNumber2.Value)
            gcTime1.TouchContextMenuScale = CSng(gcNumber2.Value)
            gcTimeSpan1.TouchContextMenuScale = CSng(gcNumber2.Value)
            gcNumber1.TouchContextMenuScale = CSng(gcNumber2.Value)
            gcComboBox1.TouchContextMenuScale = CSng(gcNumber2.Value)
            GcMaskedComboBox1.TouchContextMenuScale = CSng(gcNumber2.Value)
        End Sub

    End Class
End Namespace
